### ISSUE TITLE
player veh

### ISSUE THREAD

**Issue by Livewithrusty** [ArkSeyonet/esx_locksystem#1]
currently having a problem with the Locking portion of this on the new update player owned Vehicles don’t ever find the keys

**Comment by ArkSeyonet** [comment]
I'll check into it.

**Comment by ArkSeyonet** [comment]
I cannot replicate this issue, I purchased a vehicle, used `]` to find the keys, locked and unlocked the vehicle. Stored the vehicle in the garage, restarted the server, pulled the vehicle out of the garage, used `]` to find the keys again, and locked and unlocked the vehicle again.

**Comment by justweb1** [comment]
@Livewithrusty Can you provide more information?

**Comment by Livewithrusty** [comment]
Yes the current issue we are having  is when a player owned vehicle is pulled out of the garage some people cannot find the keys to their own personal vehicle it shows up vehicle is owned or some sort of form of that we are We are currently using esx_advancedgarage and esx_kashacters (Multi character) server that it is running on is play.newagerp.com  and if need be you can contact me directly through Discord Livewithrusty#2422

**Comment by justweb1** [comment]
If you can provide some logging statements and find out where the script is failing that would be helpful for us to fix. 

Are you using the [latest release](https://github.com/ArkSeyonet/esx_locksystem/releases/latest)?

We are currently working on a new version rewritten from almost the ground up. You may need to wait for that.
